### PR TITLE
PSR-2 Formatting

### PR DESCRIPTION
**Pull request by pmclain**
* Correct PSR-2 linter errors and notices
* Remove duplicate element in `di.xml`

Related to #22

### CODE REVIEW

**Comment by Ethan3600** [comment]
Can we remove the merge commits?

**Comment by pmclain** [comment]
yup yup

**Comment by Ethan3600** [comment]
Looks like we couldn't avoid the conflicts :angry:

**Comment by pmclain** [comment]
@Ethan3600 I rebased to fix the conflicts

File: Model/Manager.php [hunk 1]
```diff
@@ -1,130 +1,147 @@
...
+
+    public function flushCrons()
+    {
+        $jobGroups = $this->_config->getJobs();
+        foreach ($jobGroups as $groupId => $crons) {
+            $this->_cleanup($groupId);
+        }
+    }
+
+    public function dispatchCron($jobId = null, $jobCode, $schedule = null)
```
Comment (pmclain) [inline Model/Manager.php:69]: This was the only remaining item picked up by the linter/sniffer I was using. Arguments with default values should be after required arguments, but it didn't look right to have the id after the job code. Any thoughts?

Comment (Ethan3600) [reply Model/Manager.php:69]: I agree with you, the ID should take precedence. Let's let it slide this time.

**Pull request merged**